benchmarks/graphics_pipeline/main.cpp

```diff
@@ -637,18 +650,24 @@ void ProjApp::Setup()
         PPX_CHECKED_CALL(GetDevice()->CreateDescriptorPool(&poolCreateInfo, &mDescriptorPool));
     }
 
-    for (size_t i = 0; i < kAvailableShaders.size(); i++) {
-        const std::string shaderBaseName = kAvailableShaders[i];
+    uint32_t pipeline_index = 0;
+    for (size_t i = 0; i < kAvailableVsShaders.size(); i++) {
+        for (size_t j = 0; j < kAvailablePsShaders.size(); j++) {
```

Comment:
It seems like we are creating the same VS and PS shaders multiple times. Do we expect to be adding more shaders? I am curious if there a reason not to create each unique one once and then reference it

Reply:
the shaders for benchmark are supposed to be unique, we also might need to tune them in the future.